docker-compose.yml

```diff
@@ -57,6 +58,69 @@ services:
     volumes:
       - ./pg_data:/var/lib/pgsql/data
 
+  rabbit:
+      container_name: koku_rabbit
+      hostname: rabbit
+      image: rabbitmq:latest
+      environment:
+          - RABBITMQ_DEFAULT_USER=guest
+          - RABBITMQ_DEFAULT_PASS=guest
+      ports:
+          - "${RABBITMQ_PORT-5673}:5672"
+
+  koku-worker:
+      container_name: koku_worker
+      hostname: koku_worker
+      build:
+          context: .
+          dockerfile: Dockerfile-env
+      working_dir: /koku/koku
+      entrypoint: ['celery', '-A', 'koku', 'worker', '-l', 'info']
+      environment:
+        - DATABASE_ENGINE=postgresql
+        - DATABASE_NAME=${DATABASE_NAME-postgres}
+        - DATABASE_HOST=db
+        - DATABASE_PORT=5432
+        - DATABASE_USER=${DATABASE_USER-postgres}
+        - DATABASE_PASSWORD=${DATABASE_PASSWORD-postgres}
+        - RABBITMQ_HOST=rabbit
```

Comment:
Just looked over at masu and this is how it is.  Ideally it would pick up the environment variable `RABBITMQ_HOST` and default to rabbit `${RABBITMQ_HOST-rabbit}`.  In fact all of the docker-compose environment variables should probably do this.  Not a big deal if you don't want to change it here.  Main thing is we shouldn't be using the docker-compose host names in the `koku/settings.py`

Reply:
Should I also be adding the rabbitmq environment variables to .env.example?

Reply:
My only only hesitation is that the default values of setting it in the .env example can get sort of confusing.  For example.  When using docker-compose the `RABBITMQ_HOST` needs to be `rabbit` if you are using the rabbit service from docker-compose (which I can imagine would be the majority when using our compose file).  But under any other context (local, OpenShift) it really doesn't make sense.  

I wanted to make the docker-compose deployment be as standalone as possible where it basically defaults to the right values for other services in the compose yaml.  Only thing people would have to add is the AWS secrets.

That being said we could force the user to set all environment variables and put the defaults in different .env example files.  Like `.env.example.docker`, `.env.example.openshift', `.env.example.local`.  So if you're going to make that change I think pulling all defaults into separate example files would make the most sense.  I can see pro/cons on both sides of this. :)

Reply:
I'll not touch the `.env` file in that case. The defaults in the docker-compose environment are reasonable. And if there is a need to change the defaults settings in it often, we can pull out the env variables into a separate file then.